### PR TITLE
docs: Hyperlink to HTML & CSS

### PR DESCRIPTION
**Pull request by ghost**
Link to have a reference on the HTML & CSS for basic understanding.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/docs.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Tutorials and guides include downloadable example to accelerate your projects.
 ## Assumptions
 
 
-These docs assume that you are already familiar with HTML, CSS, [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript "Learn JavaScript"), 
+These docs assume that you are already familiar with [HTML](https://developer.mozilla.org/docs/Learn/HTML/Introduction_to_HTML "Learn HTML"), [CSS]()https://developer.mozilla.org/docs/Learn/CSS/First_steps "Learn CSS", [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript "Learn JavaScript"),
```
Comment (gkalpak) [inline aio/content/marketing/docs.md]: Incorrect parenthesis placement in CSS link:

```diff
-[CSS]()https://developer.mozilla.org/docs/Learn/CSS/First_steps "Learn CSS"
+[CSS](https://developer.mozilla.org/docs/Learn/CSS/First_steps "Learn CSS")
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Awesome! One typo, otherwise lgtm :+1:

File: aio/content/marketing/docs.md (hunk 1 above)
Comment (dishanfernando) [reply aio/content/marketing/docs.md]: I have done the changes and committed. please review the change.

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
Thx, @dishanfernando! Can you, please, squash the commits into one?

**Comment by dishanfernando** [comment]
Can you tell how to combine the two commits into one?

**Comment by gkalpak** [comment]
Oh, I realized you probably created the PR through GitHub's UI. In order to merge the commits you need to check them out on your computer and run some git commands on them. Let me do it for you.

If you are wondering, here is what you would need to do:

```
git clone https://github.com/dishanfernando/angular.git
cd angular
git fetch origin patch-3
git checkout -b patch-3 origin/patch-3
git reset HEAD~1
git commit --amend --no-edit
git push --force-with-lease
```

**Comment by gkalpak** [comment]
**merge-assistance:** Global approval for this docs-only change.
(Also, Igor +1'd this change in the original PR: https://github.com/angular/angular/pull/33811#discussion_r346612637)

**Comment by dishanfernando** [comment]
This is my very first contribution. Thanks for your help. Learning a lot from this. 
Will follow this method next time.

**Pull request closed without merging**